strawberryfields/backends/fockbackend/ops.py

```diff
@@ -415,6 +416,22 @@ def squeezing(r, theta, trunc, save=False, directory=None):
     return ret
 
 
+@functools.lru_cache()
+def two_mode_sqeezing(r, theta, trunc):
+    r"""The two-mode squeezing operator :math:`S_2(re^{i\theta})`.
+
+    Args:
+            r (float): two-mode squeezing magnitude
+            theta (float): two-mode squeezing phase
+            trunc (int): Fock ladder cutoff
```

Comment:
Seems to be too much indentation here, should only be 4 space indentation after the `Args:`.

If you want to test how the Strawberry Fields documentation looks locally, you can run
```bash
pip install -e doc/requirements.txt
make docs
```

That being said, I don't think this file is currently being displayed in the documentation!

Reply:
Thanks for spotting this. I'm not sure why I indented it so much. It's fixed now! Thanks for the make docs. I'll check it out.